### ISSUE TITLE
Read-Only Entities do not work

### ISSUE THREAD

**Issue by HGassner** [syssi/homeassistant-goecharger-mqtt#60]
I could intenrate go-eCharger successfully - the device and 98 entities can be found.
But only the switches are available, all read-only entities are not available and the sensors could not be read.

![image](https://user-images.githubusercontent.com/6275193/205841784-c26e460d-e016-4a49-aa95-9cc06fc565ac.png)

Do I miss something or does this just not work with the latest firmware? I'm using 054.7., the hardware number is V3.

**Comment by syssi** [comment]
You device is supported in general. Could you verify some details:

1. Make sure the `MQTT` component is loaded and conected to your MQTT broker.
2. Go to `Settings -> Devices -> MQTT -> configure` and use the `Listen to a topic` section. Do you know the topic prefix of your device? If I start listening to `/go-eCharger/072246/fhz` I see something like this:

```
Message 7 received on /go-eCharger/072246/fhz at 8:23 AM:
50.05
QoS: 0 - Retain: false
Message 6 received on /go-eCharger/072246/fhz at 8:23 AM:
49.931
QoS: 0 - Retain: false
```

**Comment by HGassner** [comment]
Thank you for your fast response! 
I just checked this. I get no message.

Perhaps there is something wrong how I've integrated the device?
The other ZigBee devices are working well - Integration is MQTT in comparison to go-eCharger (MQTT). This is the only difference I can see.

![image](https://user-images.githubusercontent.com/6275193/205890009-384c659a-145e-44a4-a5e5-2bf63baae52e.png)

**Comment by HGassner** [comment]
Just played around again, but could not find a solution.
I have 4 MQTT-Devices (ZigBee) as you can see in the screenshot of my last message.

The Mosquito Broker (MQTT) seems only to see these 4 devices. The go-eCharger has its own integration entry. Is this okay? I'm very new to all this stuff therefore, I think, that there must be an error on my side configuring the go-eCharger.

![image](https://user-images.githubusercontent.com/6275193/206416628-a26df09c-ff5d-4f27-b45d-6f9019eda5dc.png)

**Comment by syssi** [comment]
Please update your charger to `054.9` and double check the MQTT broker configuration at your Android app. What do you use here?

**Comment by HGassner** [comment]
Okay, I think I found a reason. The protocol output from the Mosquitto Broker shows an error.

> 2022-12-08 18:27:51: New connection from 192.168.0.234:58324 on port 1883.
2022-12-08 18:27:51: Client <unknown> disconnected, not authorised.

The password I've set in the Mosquitto Url in the go-E Android app is the one for my homeassistant admin.
It looks like mqtt://user:pw@192.168.x.x:1883

I saw that there were some changes in the firmware, but I'm not able to activate 0.54.9 Beta. I will keep you updated - the fault seems to be on my site - sorry for stealing your time!
